compiler-plugin/src/main/java/io/ballerina/stdlib/http/compiler/HttpResourceValidator.java

```diff
@@ -443,6 +411,44 @@ public static void extractInputParamTypeAndValidate(SyntaxNodeAnalysisContext ct
         }
     }
 
+    private static boolean isValidPayloadParamType(TypeSymbol typeDescriptor, SyntaxNodeAnalysisContext ctx,
```

Comment:
Do we allow user to define type as `anydata` here?

Reply:
Added via ce6b9f09e290feda3bf35ec83ff65536a0fb837f